multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -23,6 +23,9 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
```

Comment:
For whatever reason, this doesn't work. Not sure how to override the envs correctly here.